api/internal/conf/conf.go

```diff
@@ -283,6 +298,14 @@ func initAuthentication(conf Authentication) {
 	}
 }
 
+func initOidc(conf Oidc) {
+	OidcConf = conf
+
+	if OidcConf.Secret == "secret" {
+		OidcConf.Secret = utils.GetFlakeUidStr()
```

Comment:
I think we should not set a random string to Secret. we could add a warning for user.

Reply:
After thinking twice, I think the `secret` is not necessary, I will delete it in the next commit.